internal/resource/config/serversettings/systemkeys/server_settings_system_keys_resource.go

```diff
@@ -52,7 +52,8 @@ type serverSettingsSystemKeysResource struct {
 // GetSchema defines the schema for the resource.
 func (r *serverSettingsSystemKeysResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
 	schema := schema.Schema{
-		Description: "Manages the system encryption keys.",
+		Description:        "Manages the system encryption keys.",
+		DeprecationMessage: "This resource is deprecated and will be removed in a future release.",
```

Comment:
We should describe which resource to use instead in this message, as well as updating the docs and the changelog to mention the deprecation.